### DISCUSSION TITLE
SwissGerman Crawler

### DISCUSSION THREAD

**Discussion by esthicodes** [Estheryu991/SwissGerman_Dictionary#45] [Ideas]
// Challenge:
• "Wie komme ich an die News?"
• Crawler: Informationen von irgendwo holen

• "Wie komme ich an die News gsi?"
• Crawler: Informationen von irgendwo hole

- Comment obtenir les nouvelles" ?
- Crawler : aller chercher des informations quelque part

**Comment by esthicodes** [comment]
Ideen:
• 

- [ ] News-Aggregator mit News von verschiedenen Quellen

• 

- [ ] Interface mit Zusammenfassungen

- [ ]  News-Kurator, welcher nur über das

- [ ] Wichtigste benachrichtigt

- [ ] Politischer Bias oder Wahrheitsgehalt anzeigen

**Comment by esthicodes** [comment]
How can API be connected with [Crawler?](https://github.com/yasserg/crawler4j)

[Twitter Crawler](https://github.com/twitterdev/Twitter-API-v2-sample-code)
[API](https://github.com/scrapy/scrapy)
[Crawling](https://github.com/scrapy/scrapy)
[Bias](https://github.com/wandb/wandb)
https://github.com/parthasm/K-Means-Clusters-Words/blob/master/K_Means_Cluster.py

Semantische Haupt Thema 
Zuteilung mit Political Handlung 
Twitter Crawler
Interface 

Machine Learning Algorithms to make Headlines, URL, assignen kann
Topics ersetzen 

NLP 
Bias Allocator



Google News an stadt 
alles  News Article 
Idee> =

wenn du die Szstem anreufst, hast du Liste mit Trending articles, 
dann crawled 
die Twitter crawled. 

Zusammenfassung>> Wie tut wir das die Artikle ist die Topics anzeige
Die Liste haben wir die Ukraine Artikel. 

Topics sind da. 
Welche Topic haetten wir?

Liste von Artikel. 

Wo ist die Elemente? 

Die Groesse Size  100 Gleichen Thema Russiche Addresse, betreffen gleichen Thema. weiss nicht die Ansicht die Verschiedene Thema 

Verschiedene Werte, 

wie machen wir die Oberbegriff 

Titel das ist verschiedene Idee zusammenbauen. 

Ukriane. 

Programmiere mit Durian. 

Was sind die Trending gsi?

**Comment by generated** [comment]
idee find your classical doppelgänger - face recognition -> search classical paintings from archive basel kunstmuseum
